tifeatures/factory.py

```diff
@@ -74,7 +82,7 @@ def create_html_response(
                 "title": "",
             },
             "crumbs": crumbs,
-            "json_url": str(request.url).replace("f=html", "f=json"),
+            "url": str(request.url),
```

Comment:
I'm passing the url and then add `f=geosjon` or `f=json` in the template directly